### PR TITLE
Add unit test for `_fixImpliedMeridiemOfChronoResult`

### PR DESCRIPTION
**Pull request by arthurshir**
- Test implied meridian text adjustment output
- Test implied meridian date output
- Add export for `_fixImpliedMeridiemOfChronoResult` function
- Update the TIME_ZONE constant to be consistent with naming convention of other files

### CODE REVIEW

File: tst/nlp/parseNonRecurringSingleDate.test.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
 const parseNonSingleRecurringDate = require('../../nlp/parseNonRecurringSingleDate.js'),
+    moment = require('moment-timezone'),
     timemachine = require("../../timemachine.js");
 
+const TIME_ZONE = "America/Los_Angeles";
```
Comment (bubakazouba) [inline tst/nlp/parseNonRecurringSingleDate.test.js]: TIMEZONE instead of TIME_ZONE for consistency over the repo

File: tst/nlp/parseNonRecurringSingleDate.test.js [hunk 2]
```diff
@@ -48,4 +51,62 @@ describe("_convertOnTimetoAtTime", () => {
             expect(parseNonSingleRecurringDate._convertOnTimetoAtTime(key)).toEqual(map[key]);
         }
     });
+});
+
+describe("_fixImpliedMeridiemOfChronoResult", () => {
```
Comment (bubakazouba) [inline tst/nlp/parseNonRecurringSingleDate.test.js]: lets add a couple of tests where the function takes the full date/time and not just the time
and another edge case for when the function takes a date without at time

**Comment by arthurshir** [comment]
Merging into development instead of master
https://github.com/bubakazouba/remindmebot/pull/2
